### ISSUE TITLE
Shrink Pro (with live preview) - Add option to adjust sound volume

### ISSUE THREAD

**Issue by gileli121** [gileli121/WindowTop#28]
A user sent the following story:

**Hi, purchased the Pro version and love it. What a great, useful tool you created. Especially the Live Prevew.. wow! Thank you.

Feature request:
- I often have 2 or 3 "shrunk" windows that have sound (eg. YouTube, Live TV, etc.).
- Via Live Preview I might see something exciting in one window and have to maximise the window that already has the sound, turn sound off, re-shrink, maximise window I want sound, turn on sound, re-shrink.
- Wondering it's at all possible for WindowTop to control sound On/Off using WindowTop Right Click menu? Perhaps one step better, if sound turned "on" in one shrunk window, WindowTop automatically turns off sound in all other shrunk windows?

Not sure if it's even possible for you to do this with software, but would be very useful.

Thank you for your consideration.**

The idea is to add to the live preview also an option to control the sound of the live preview box.
This idea is very good and I thought about it myself while using the feature.
I will try to find the time to add this also when I continue to develop the product.

**Comment by gileli121** [comment]
Functionality implemented.

Old toolbar:
![image](https://user-images.githubusercontent.com/17680514/97090954-d5771000-1640-11eb-976c-b5cefb0a7699.png)
![image](https://user-images.githubusercontent.com/17680514/97091335-fa20b700-1643-11eb-8915-a58a0f7af5af.png)

Modern toolbar will be implemented next

**Comment by gileli121** [comment]
After investigation, it is doable, but there is a limitation:
If I use the Windows API to control sound volume, for example of google chrome, and you have 2 videos playing at the same time and, you lower the volume to only one of the videos, it will lower the volume to all windows of google chrome because the API allows you to change volume level **per process and not per window**.
What we need is to be allowed to change the volume level **per window**. 
The API in Windows will not allow it, and it also can't. 
This type of control can be done only from within the process itself.

**Bottom line:
It is not doable if we want to contol volume per window (and this is exactly what we should do).
So it can't be done...

What can be done instead is maybe somehow develop an option to click inside the window while shrink mode enabled
so you will able to access the window while it is on shrink mode...

I will investigate how it can be done

Closing this issue
**
